core/src/main/scala/org/apache/spark/util/Benchmark.scala

```diff
@@ -91,16 +92,31 @@ private[spark] object Benchmark {
    * This should return something like "Intel(R) Core(TM) i7-4870HQ CPU @ 2.50GHz"
```

Comment:
nah i don't think we need to do those. These are meant as a snapshot for reference.